packages/create-just/src/commands/initCommand.ts

```diff
@@ -107,12 +111,14 @@ Please make sure you have git installed and then issue the following:
 
   logger.info('All Set!');
 
-  showNextSteps(argv, stackName);
+  showNextSteps(argv, stackName, stackPath!);
 }
 
-function showNextSteps(argv: any, stackName: string) {
-  logger.info(
-    prettyPrintMarkdown(`
+function showNextSteps(argv: any, stackName: string, stackPath: string) {
+  const postInitPath = path.join(stackPath, 'PostInit.md');
```

Comment:
Can this be named something else? Some choices:
* post-create-just.md
* post-generate.md
* post-create.md

Or just do no casing:

* postcreate.md
* postgen.md
* postgenerate.md